src/App.tsx

```diff
@@ -15,48 +17,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  name = 'name',
+  length = 'length',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sort: string,
+  reverse: boolean,
+): string[] {
+  const preparedGoods = [...goods];
+
+  if (sort !== null) {
```

Comment:
pls see [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)
```suggestion
  if (!!sort) {
```